ai/prompt_context/html_docs_context.txt

```diff
@@ -0,0 +1,95 @@
+# ../../site/404.html
+# ../../site/demo/index.html
+# ../../site/goals/index.html
+../../site/faq/index.html
+# ../../site/codelab/index.html
+# ../../site/codelab/4/index.html
+# ../../site/codelab/3/index.html
+# ../../site/codelab/2/index.html
+# ../../site/codelab/5/index.html
+# ../../site/internal/publishing/index.html
+# ../../site/internal/modes/index.html
+# ../../site/internal/development/index.html
+# ../../site/internal/ci/index.html
+# ../../site/internal/contributing/index.html
+# ../../site/internal/new-component/index.html
+# ../../site/internal/testing/index.html
+# ../../site/internal/codespaces/index.html
+# ../../site/internal/architecture/index.html
+# ../../site/internal/vs-code-remote-container/index.html
+# ../../site/internal/type-checking/index.html
+# ../../site/internal/hot-reload/index.html
+# ../../site/internal/toolchain/index.html
+# ../../site/blog/index.html
+# ../../site/blog/2024/05/13/why-mesop/index.html
+# ../../site/blog/2024/07/12/is-mesop--web-components-the-cure-to-front-end-fatigue/index.html
+# ../../site/blog/2024/01/12/visual-editor/index.html
+# ../../site/blog/2023/12/25/hello-mesop/index.html
+# ../../site/blog/archive/2024/index.html
+# ../../site/blog/archive/2023/index.html
+../../site/web-components/index.html
+../../site/web-components/troubleshooting/index.html
+../../site/web-components/quickstart/index.html
+../../site/web-components/api/index.html
+../../site/components/index.html
+../../site/components/uploader/index.html
+../../site/components/video/index.html
+../../site/components/tooltip/index.html
+../../site/components/sidenav/index.html
+../../site/components/box/index.html
+../../site/components/radio/index.html
+../../site/components/plot/index.html
+../../site/components/autocomplete/index.html
+../../site/components/embed/index.html
+../../site/components/chat/index.html
+../../site/components/input/index.html
+../../site/components/markdown/index.html
+../../site/components/progress-bar/index.html
+../../site/components/slide-toggle/index.html
+../../site/components/text-to-image/index.html
+../../site/components/checkbox/index.html
+../../site/components/code/index.html
+../../site/components/slider/index.html
+../../site/components/html/index.html
+../../site/components/progress-spinner/index.html
+../../site/components/image/index.html
+../../site/components/text-to-text/index.html
+../../site/components/audio/index.html
+../../site/components/textarea/index.html
+../../site/components/button/index.html
+../../site/components/link/index.html
+../../site/components/table/index.html
+../../site/components/divider/index.html
+../../site/components/text/index.html
+../../site/components/select/index.html
+../../site/components/icon/index.html
+../../site/components/badge/index.html
+../../site/getting-started/installing/index.html
+../../site/getting-started/quickstart/index.html
+../../site/getting-started/core-concepts/index.html
+../../site/theme/partials/search.html
```

Comment:
What is are these partials page?

Reply:
it's for customizing the docs ui - i've commented them out.